### ISSUE TITLE
IllegalArgumentException

### ISSUE THREAD

**Issue by Yoghurt4C** [natanfudge/SpatialCrafting#8]
Caused by: java.lang.IllegalArgumentException: Packet with serializer spatialcrafting.Packets$AssignMultiblockState$$serializer@579c7d7d was not registered!
        at Not Enough Crashes deobfuscated stack trace.(1.15+build.2) ~[?:?]
        at fabricktx.api.NetworkingKt.sendPacket(NetworkingKt.java:56) ~[74652e88-cd97-42aa-9425-067571ef924e.jar:?]
        at fabricktx.api.NetworkingKt.sendPacket(NetworkingKt.java:47) ~[74652e88-cd97-42aa-9425-067571ef924e.jar:?]
        at spatialcrafting.crafter.CrafterMultiblockFormationKt.createMultiblockFromServer(CrafterMultiblockFormationKt.java:37) ~[spatialcrafting-1.4.1+1.15.1-modpack.jar:?]
        at spatialcrafting.crafter.CrafterMultiblockFormationKt.attemptToFormMultiblock(CrafterMultiblockFormationKt.java:21) ~[spatialcrafting-1.4.1+1.15.1-modpack.jar:?]
        at spatialcrafting.crafter.CrafterPieceBlock.onPlaced(CrafterPieceBlock.java:144) ~[spatialcrafting-1.4.1+1.15.1-modpack.jar:?]
        at net.minecraft.item.BlockItem.place(BlockItem:81) ~[?:?]

spatialcrafting 1.4.1

**Comment by Yoghurt4C** [comment]
![2019-12-25_19 24 08](https://user-images.githubusercontent.com/11313954/71448376-33e70f80-274c-11ea-86af-5641f3882124.png)

**Comment by Yoghurt4C** [comment]
```
Caused by: java.lang.IllegalStateException: BlockEntity at location (x = -324,y = 63,z = -2588) is not a spatialcrafting.crafter.CrafterPieceEntity as expected. Rather, there is no block entity, and at the position exists a Block{minecraft:moving_piston}. This happened at the Server.
        at Not Enough Crashes deobfuscated stack trace.(1.15+build.2) ~[?:?]
        at fabricktx.impl.MinecraftExtensionsImplKt.throwDiagnosticMessage(MinecraftExtensionsImplKt.java:19) ~[3123c672-2537-49a3-828b-aa1177549a6e.jar:?]
        at spatialcrafting.crafter.CrafterPieceBlockKt.getCrafterEntity(CrafterPieceBlockKt.java:151) ~[spatialcrafting-1.4.1+1.15.1-modpack.jar:?]
        at spatialcrafting.crafter.CrafterPieceBlock.onBlockRemoved(CrafterPieceBlock.java:92) ~[spatialcrafting-1.4.1+1.15.1-modpack.jar:?]
        at net.minecraft.block.BlockState.onBlockRemoved(BlockState:259) ~[?:?]
```

**Comment by natanfudge** [comment]
Thank you for the report kind user

**Comment by natanfudge** [comment]
Fixed as of 1.4.2
